### PR TITLE
finalized clean architecture implementation

### PR DESCRIPTION
**Pull request by samidc1992**

### CODE REVIEW

File: routes/sports.js [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
-        }
-        else {
-            // User already exists in database
-            res.json({ result: false, error: 'Sport already exists' });
-        }
-    })
+    };
+
+    const { label } = req.body;
+    const sportName = label[0].toUpperCase() + label.slice(1).toLowerCase();
```
Comment (aymericsecret) [inline routes/sports.js:15]: This could be directly formatted in the frontend ?

File: services/Sport.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const SportEntity = require('../entities/Sport');
+const Sport = require('../models/sports');
+
+const getAllSports = async() => {
+    const sportsData = await SportEntity.findAllSports();
+    sportsNames = sportsData.map(sport => sport.label);
+    return sportsNames;
```
Comment (aymericsecret) [inline services/Sport.js]:
```suggestion
    return sportsData.map(sport => sport.label);
```
sportsNames was not even declared

File: services/Sport.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+const findSportByName = async(sportName) => {
+    const sportData = await SportEntity.findSportBySportName(sportName);
+    return sportData;
```
Comment (aymericsecret) [inline services/Sport.js]:
```suggestion
    return SportEntity.findSportBySportName(sportName);
```
Maybe in an otherpr you can simplify all those functions

File: utils/uploadPictureToCloud.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+const cloudinary = require('cloudinary').v2;
+const uniqid = require('uniqid');
+const fs = require('fs');
+
+const uploadPictureToCloudinary = async(files) => {
+
+    const photoPath = `./tmp/${uniqid()}.jpg`;
+    const resultMove = await files.mv(photoPath);
+
+    if(!resultMove) {
```
Comment (aymericsecret) [inline utils/uploadPictureToCloud.js]: Maybe you should add a comment here to explain why you're expecting a nullable value from .mv()

File: routes/sports.js (hunk 1 above)
Comment (samidc1992) [reply routes/sports.js:15]: no, as this controller is only used by developers to create new sports

File: utils/uploadPictureToCloud.js (hunk 4 above)
Comment (samidc1992) [reply utils/uploadPictureToCloud.js]: I will do!

**Pull request merged**